README.md

```diff
@@ -186,6 +186,14 @@ sortable('.sortable', {
   maxItems: 3 // Defaults to 0 (no limit)
 });
 ```
+### acceptFrom
+Use the `acceptFrom` option to restrict the which sortable's items will be accepted by this sortable. acceptForm accepts a space separeted list of selectors or '' to accept nothing.
+
+``` javascript
+sortable('.sortable', {
+  acceptForm: '.selector' // Defaults to false (defaults to connectWith behavior)
```

Comment:
My initial thoughts were comma separated. I'll change it to comma separated.

Reply:
👍

Reply:
But the comment would have to be `// Defaults to null` correct? And I think we should remove the `(defaults to connectWith behavior)` part here, as it might be confusing.